BraveWallet/Preview Content/TestEthRpcController.swift

```diff
@@ -33,7 +33,7 @@ class TestEthJsonRpcController: NSObject, BraveWalletEthJsonRpcController {
   }
   
   func balance(_ address: String, completion: @escaping (Bool, String) -> Void) {
-    completion(true, "10")
+    completion(true, "0x13e25e19dc20ba7")
```

Comment:
Can you comment what why these values are used in this completion block ?

Reply:
resolved at [c12c8d7](https://github.com/brave/brave-ios/pull/4741/commits/c12c8d7fae8c891efc4b8db300e7e651cea7c092)